content/articles/how-to-implement-structured-logging-in-adonisjs/index.md

```diff
@@ -0,0 +1,261 @@
+## How to implement Structured Logging in Adonis.js
+Logs are among the essential tools for application developers during the debugging process and for analysts monitoring applications in the production environment.
+
+Typical logs are textual formats that are easily readable by humans but very difficult for machines to read and process.
+
+Assuming you want to use automated processing to investigate log files, or you want to use algorithms to categorize, index, and search through log files based on specific parameters, typical logs will not work as expected.
+
+That is where structured logging becomes useful. Structured logging is a way to record the information in a way that is easy for machines to process.
+
+This article will show the best practice in implementing structured logging in Adonis.js 5, discussing in detail, the best ways involved in implementing a scalable logging system in your RESTFUL API. 
```

Comment:
Here are some of the major issues I encountered while going through a prelim review on your article.

1. Take note of your spelling. Be consistent. If you choose to use for example the word "Adonis.js", you should use it consistently throughout your tutorial. Don't interchange it with "Adonis" or any other spelling of the word. eg. "AdonisJS".
2. Inconsistent capitalization. There were many case of capitalized word in a sentence. For example, the words "structured logging" were most of the time capitalized in sentense. ie. "Structured Logging". Please see [this thesaurus page](https://www.thesaurus.com/e/grammar/when-to-capitalize-words/) on capitalization rules.


> 🔷 **Please take note of the above in future tutorials.**